README.md

```diff
@@ -668,6 +684,96 @@ Errors:
 
 - `ExpectationFalse`: If the value of the Error Register is not equal to the parameter.
 
+### `QueryPallet`
+
+Query the existence of a particular pallet type.
+
+- `name: Vec<u8>`: The name of the pallet to query.
+- `query_id: QueryId`: The value to make the returned message identifiable with this query.
+- `max_response_weight: Weight`: The value to be used for the `max_weight` field of the `QueryResponse` message.
+
+Sends a `QueryResponse` to Origin whose data field `PalletsInfo` containing the information of all pallets on the local chain whose name is equal to `name`. This is empty in the case that the local chain is not based on Substrate Frame.
```

Comment:
is `Substrate Frame.` is must to support SCALE dispatch? That stuff is quite generic too enum(enum(origin, data)) to be be little be broader than only Substate Frame (i thin of it easy to have WS with JSON RCP which parses SCALE with no Substete Frame)